README.md

```diff
@@ -15,128 +15,73 @@ for an independent review.
 
 Git Town provides these additional Git commands:
 
-**Development Workflow**
+#### Typical development commands
 
-- [git town hack](/documentation/commands/hack.md) - cuts a new up-to-date
-  feature branch off the main branch
-- [git town sync](/documentation/commands/sync.md) - updates the current branch
-  with all ongoing changes
-- [git town new-pull-request](/documentation/commands/new-pull-request.md) -
+- [git town hack](https://www.git-town.com/commands/hack.html) - cuts a new
```

Comment:
I'm moving the documentation of all commands to the website. The website is now for end-user facing documentation, and the GitHub readme files document how to contribute to Git Town.